tests/exchange/binance_ohlcv_compare.py

```diff
@@ -0,0 +1,136 @@
+"""
+This file is meant to test the OHLCV download performance between
+from rest API and from data.binance.vision
```

Comment:
do we intend to keep this file? if yes, then maybe it should be moved to scripts.

I guess it's however only here for performance testing ...?